Makefile.rules

```diff
@@ -193,12 +193,12 @@ GAP_CPPFLAGS += -I$(builddir)/build
 SYSINFO_CPPFLAGS += -I$(abs_builddir)/build
 
 # The other source files are relative to the srcdir
-GAP_CPPFLAGS += -I$(srcdir)/src
-SYSINFO_CPPFLAGS += -I$(abs_srcdir)/src
+GAP_CPPFLAGS += -iquote$(srcdir)/src
```

Comment:
Why `-iquote` ?

Reply:
OK, fair enough. Can you please add that as a comment explaining this rationale, so that just in case someone has trouble with this line in the future they know why it  uses `-iquote` instead of `-I` ?